[     1.595][MainProcess][INFO] Parsing specification...
[     1.600][MainProcess][DEBUG] Reading input files...
[     1.606][MainProcess][INFO] Inferred data types for table tests-examples/scythe/recent_posts/tables/033_1.csv: ['int64', 'int64', 'int64']
[     1.644][MainProcess][INFO] Inferred data types for table tests-examples/scythe/recent_posts/tables/033_2.csv: ['int64', 'int64', 'int64', 'object']
[     1.652][MainProcess][INFO] Inferred data types for table tests-examples/scythe/recent_posts/tables/033_o.csv: ['int64', 'int64', 'int64', 'int64', 'int64']
[     1.683][MainProcess][DEBUG] Generating DSL...
[     1.705][MainProcess][INFO] Creating 2 processes
[     1.715][MainProcess][DEBUG] Increasing generator loc to 3
[     1.718][MainProcess][DEBUG] Increasing generator loc to 4
[     1.742][MainProcess][DEBUG] Increasing generator loc to 5
[     1.951][MainProcess][DEBUG] Increasing generator loc to 6
[     4.081][MainProcess][DEBUG] Increasing generator loc to 7
[    25.859][MainProcess][DEBUG] Increasing generator loc to 8
[   247.771][cube-solver-0][DEBUG] Initialising process for 8 lines of code.
Traceback (most recent call last):
  File "./cubes.py", line 95, in <module>
    main()
  File "./cubes.py", line 87, in main
    program = synthesizer.synthesize()
  File "/home/rmbb/CUBES/squares/parallel_synthesizer.py", line 250, in synthesize
    cube = next(generator)
  File "/home/rmbb/CUBES/squares/dsl/dc.py", line 131, in __next__
    while not self.line_stack[-1]:
IndexError: list index out of range
